features/shipping/managing_shipping_methods/adding_shipping_method.feature

```diff
@@ -13,6 +13,7 @@ Feature: Adding a new shipping method
     Scenario: Adding a new shipping method with flat rate per shipment
         Given I want to create a new shipping method
         When I specify its code as "FED_EX_CARRIER"
+        And I specify its position as 0
```

Comment:
It checks that position can be set. Position 0 is the first element, empty position equals -1 which equals the last element.